### PR TITLE
docs: Clarify rollback procedure for systemd-installed gateways

### PR DESCRIPTION
**Pull request by jamilbk**
* We'll be bumping the versions going forward, so instructions need a bit of clarification.
* Added rollback instructions for systemd Gateway upgrades that have gone awry.

### CODE REVIEW

File: website/src/app/kb/administer/upgrading/readme.mdx [hunk 1]
```diff
@@ -73,13 +51,52 @@ necessary to upgrade the Gateway on service start. Simply move the old binary
...
+Please
+[file a GitHub issue](https://www.github.com/firezone/firezone/issues/new/choose)
+if you consistently encounter issues with the upgrade process.
+
 </TabsItem>
 </TabsGroup>
 
+### Verification
+
+After running the upgrade, you can verify the Gateway is r:wqaunning the latest
```
Comment (bmanifold) [inline website/src/app/kb/administer/upgrading/readme.mdx]:
```suggestion
After running the upgrade, you can verify the Gateway is running the latest
```

File: website/src/app/kb/administer/upgrading/readme.mdx [hunk 2]
```diff
@@ -73,13 +51,52 @@ necessary to upgrade the Gateway on service start. Simply move the old binary
...
+version by checking what version it is reporting in the admin portal.
```
Comment (bmanifold) [inline website/src/app/kb/administer/upgrading/readme.mdx]: What do you think about making this a little more concise?

```suggestion
After upgrading, verify the Gateway's version in the admin portal to ensure it's up to date.
```

Comment (jamilbk) [reply website/src/app/kb/administer/upgrading/readme.mdx]: Updated

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**